src/Languages/Properties/ListProperties.ts

```diff
@@ -15,6 +20,11 @@ export class ListProperties {
     public className: string;
 
     /**
+     * Adds two or more lists together
```

Comment:
> Adds two or more lists together

Inaccurate. You only have tests showing this working for two lists (and I don't think all languages support multiple lists).